pages/linux/wine.md

```diff
@@ -0,0 +1,32 @@
+# wine
+
+> Run Windows programs on Unix.
+> More information: <https://wiki.winehq.org/List_of_Commands>.
+
+- Run a program with args:
+
+`wine {{program}} {{program args}}`
```

Comment:
I think we should have the msiexec example here. And remove all the examples for cmd, taskmgr. All of them are the same repetition of a program with/without args. Typically, we do not use extra examples (we only have a space of 8!) to show the same thing.

Reply:
I included those 4 sub-programs (cmd, taskmgr, msiexec and uninstaller), because i believe they are the most important, the most helpful of all. I believe that with those examples, the user has a good knowledge, for a basic usage of the tool. At least from my experience with the tool, those are all i ever need to remember, for anything i wanted to do. Although, if you find the page excessive, i agree that cmd, taskmgr  and winecfg can be omitted. But i believe uninstaller should stay, because  the cmd and tasmgr are referring to the real Windows cmd.exe and taskmgr.exe under System32 but the "Add or remove programs wizard" utility is appwiz.cpl so its a bit confusing as it is replaced with the uninstaller designation in wine.

Reply:
included those 4 sub-programs (cmd, taskmgr, msiexec and uninstaller), because i believe they are the most important, the most helpful of all. I believe that with those examples, the user has a good knowledge, for a basic usage of the tool. At least from my experience with the tool, those are all i ever need to remember, for anything i wanted to do. Although, if you find the page excessive, i agree that cmd, taskmgr and winecfg can be omitted. But i believe uninstaller should stay, because the cmd and tasmgr are referring to the real Windows cmd.exe and taskmgr.exe under System32 but the "Add or remove programs wizard" utility is appwiz.cpl so its a bit confusing as it is replaced with the uninstaller designation in wine.

Reply:
> Although, if you find the page excessive, i agree that cmd, taskmgr and winecfg can be omitted.

Sure let's do that.

And can we also use `cmd` with some args as an example for "Run a program with args" ? That should remove one more example. Then I am good with this.

Reply:
Oh yes, good suggestion.

Reply:
@icsd12015 - Not sure if you have fixed this or yet to do it, but I don't see the change that I requested. Please let me know if you need clarification.

Reply:
I thought that my last [commit ](https://github.com/tldr-pages/tldr/blob/cef11004eaa32e3cf98ee17871a0de292be4b295/pages/linux/wine.md) fulfilled your request.  If you not please do specify why.

Reply:
Sure, I wanted:

> And can we also use cmd with some args as an example for "Run a program with args" ?

I still see this: `wine {{program}} {{program args}}`

Reply:
Sorry i made the correct pr about this issue. I didn't understand it at first.